### PR TITLE
feature-erms-2405

### PR DESCRIPTION
**Pull request by croqueGrec09**
as of ERMS-2405, new subscription properties have been added to subscription copy functions

### CODE REVIEW

File: app/grails-app/views/templates/subscription/_copyPackagesAndIEs.gsp [hunk 1]
```diff
@@ -42,19 +42,39 @@
...
                             <div data-pkgoid="${genericOIDService.getOID(sp)}" class="la-copyPack-item">
-                                    <label>
-                                        <i class="gift icon"></i>
-                                        <g:link controller="package" action="show" target="_blank" id="${sp.pkg?.id}">${sp?.pkg?.name}</g:link>
-                                        <semui:debugInfo>PkgId: ${sp.pkg?.id}</semui:debugInfo>
-                                        <g:if test="${sp.pkg?.contentProvider}">(${sp.pkg?.contentProvider?.name})</g:if>
-                                    </label>
+                                <label>
+                                    <i class="gift icon"></i>
+                                    <g:link controller="package" action="show" target="_blank" id="${sp.pkg?.id}">${sp?.pkg?.name}</g:link>
```
Comment (klodav) [inline app/grails-app/views/templates/subscription/_copyPackagesAndIEs.gsp:53]: sp?.pkg?.name

Comment (croqueGrec09) [reply app/grails-app/views/templates/subscription/_copyPackagesAndIEs.gsp:53]: It was not me!!! :D

File: app/grails-app/views/templates/subscription/_copyElements.gsp [hunk 2]
```diff
@@ -78,8 +78,216 @@
                         </g:if>
                     </td>
                 </tr>
+                <tr>
+                    <td name="subscription.takeStatus.source">
+                        <div>
+                            <b><i class="ellipsis vertical icon"></i>${message(code: 'subscription.status.label')}:</b>
+                            ${sourceSubscription.status?.getI10n('value')}
```
Comment (klodav) [inline app/grails-app/views/templates/subscription/_copyElements.gsp:85]: file-wide inconsistent use of sourceSubscription.x and sourceSubscription?.x -> looks questionable

Comment (croqueGrec09) [reply app/grails-app/views/templates/subscription/_copyElements.gsp:85]: It was not me who built the feature, so actually, it is not my task either.

**Pull request merged**